### PR TITLE
feat: Add GnoStudio connect link

### PR DESCRIPTION
**Pull request by jinoosss**
# Descriptions

Based on issue #94 ,
Link to an interactive page in Gnostudio.

- Define a URL template for GnoStudio
- Add a link button to the Realm detail page

### CODE REVIEW

**Review by ilgooz — COMMENTED** [review]
❤️

**Comment by dongwon8247** [comment]
@jinoosss Can you add a tooltip on the functions with "Click to try in GnoStudio"? Refer to [this Figma](https://www.figma.com/design/X6L1Ycel6vt6Nc27OnqFYm?node-id=5002-31036#803537199), please!

File: src/pages/realms/details.tsx [hunk 1]
```diff
@@ -111,11 +146,17 @@ const RealmsDetails = ({path}: RealmsDetailsPageProps) => {
...
-                    <Text type="p4" color="white">
-                      {v}
-                    </Text>
+                {realm.funcs.map((v: string, index: number) => (
+                  <Badge
+                    className="link"
+                    key={index}
+                    type="blue"
+                    onClick={() => moveGnoStudioViewRealmFunction(v)}>
+                    <Tooltip className="tooltip" content="Try in GnoStudio">
```
Comment (dongwon8247) [inline src/pages/realms/details.tsx:155]: Can you change the text to "Click to try in GnoStudio"?

**Review by dongwon8247 — APPROVED** [review]
Good job!

**Review by dongwon8247 — APPROVED** [review]

**Pull request merged**